src/main/java/com/stripe/model/Topup.java

```diff
@@ -0,0 +1,199 @@
+        return currency;
+    }
+
+    public void setCurrency(String currency) {
+        this.currency = currency;
+    }
+
+    @Override
+    public Map<String, String> getMetadata() {
+        return metadata;
+    }
+
+    public void setMetadata(Map<String, String> metadata) {
+        this.metadata = metadata;
+    }
+
+    public Source getSource() {
+        return source;
+    }
+
+    public void setSource(Source source) {
+        this.source = source;
+    }
+
+    public String getStatus() {
+        return status;
+    }
+
+    public void setStatus(String status) {
+        this.status = status;
+    }
+
+    public String getBalanceTransaction() {
+        return balanceTransaction;
+    }
+
+    public void setBalanceTransaction(String balanceTransaction) {
+        this.balanceTransaction = balanceTransaction;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    public String getStatementDescriptor() {
+        return statementDescriptor;
+    }
+
+    public void setStatementDescriptor(String statementDescriptor) {
+        this.statementDescriptor = statementDescriptor;
+    }
+
+    public Long getExpectedAvailabilityDate() {
+        return expectedAvailabilityDate;
+    }
+
+    public void setExpectedAvailabilityDate(Long expectedAvailabilityDate) {
+        this.expectedAvailabilityDate = expectedAvailabilityDate;
+    }
+
+    public String getFailureCode() {
+        return failureCode;
+    }
+
+    public void setFailureCode(String failureCode) {
+        this.failureCode = failureCode;
+    }
+
+    public String getFailureMessage() {
+        return failureMessage;
+    }
+
+    public void setFailureMessage(String failureMessage) {
+        this.failureMessage = failureMessage;
+    }
+
+    public static Topup create(Map<String, Object> params)
+            throws AuthenticationException, InvalidRequestException,
+            APIConnectionException, CardException, APIException {
+        return create(params, null);
+    }
+
+    public static Topup create(Map<String, Object> params, RequestOptions options)
+            throws AuthenticationException, InvalidRequestException,
+            APIConnectionException, CardException, APIException {
+        return request(RequestMethod.POST, classURL(Topup.class), params, Topup.class, options);
+    }
+
+    public static Topup retrieve(String id) throws AuthenticationException,
+            InvalidRequestException, APIConnectionException, CardException,
+            APIException {
+        return retrieve(id, null);
+    }
+
+    public static Topup retrieve(String id, RequestOptions options)
+            throws AuthenticationException, InvalidRequestException,
+            APIConnectionException, CardException, APIException {
+        return request(RequestMethod.GET, instanceURL(Topup.class, id), null, Topup.class, options);
```

Comment:
Can this one be implemented using the below method:
<pre>
public static Topup retrieve(String id, Map<String, Object> params, RequestOptions options);
</pre>
?

Reply:
Fixed in 18f6b86